Makefile

```diff
@@ -99,10 +99,10 @@ ps2ftpd_irx.s: oldlibs/ps2ftpd/bin/ps2ftpd.irx
 ps2atad_irx.s: $(PS2SDK)/iop/irx/ps2atad.irx
 	$(BIN2S) $< $@ ps2atad_irx
 
-ps2hdd_irx.s: $(PS2SDK)/iop/irx/ps2hdd-osd.irx
+ps2hdd_irx.s: $(PS2SDK)/iop/irx/ps2hdd-xosd.irx
 	$(BIN2S) $< $@ ps2hdd_irx
 
-ps2fs_irx.s: $(PS2SDK)/iop/irx/ps2fs.irx
+ps2fs_irx.s: $(PS2SDK)/iop/irx/ps2fs-xosd.irx
```

Comment:
@uyjulian theres just case sensitivity issue here I think, ps2fs-Xosd.irx (hdd irx is lower case though)

Reply:
PS2SDK should be edited as well cause we now have:
`ps2hdd-xosd.irx`
but
`ps2fs-Xosd.irx`

Reply:
The following should resolve the capitalization issue: https://github.com/ps2dev/ps2sdk/pull/209